packages/pandas/1.5.0/build.py

```diff
@@ -0,0 +1,10 @@
+from builder import build
+
+build.build_package(
+    source=build.github_source(
+        org='pandas-dev',
+        repo='pandas',
+        tag='v1.5.0'),
+    setup_py_commands=['build_ext', 'bdist_wheel'],
```

Comment:
Are these used by `args_for_command()`? Will different packages have different values?

Reply:
Yeah. I think most packages will have these but it's certainly possible that some will do something else.